bentoml/_internal/bento/bento.py

```diff
@@ -131,32 +159,12 @@ def create(
         bento_fs = fs.open_fs(f"temp://bentoml_bento_{svc.name}")
         ctx_fs = fs.open_fs(build_ctx)
 
-        model_tags = build_config.additional_models
+        model_tags = build_config.additional_models or []
```

Comment:
there's no need for `or []` here, see BuildConfig's `with_defaults`

Reply:
![image](https://user-images.githubusercontent.com/1206493/144705560-9bbc3fa3-ad66-4561-866f-5bbb77437dc6.png)
the type is `Optional` so we must check it

Reply:
yes, although the expectation is this will never be `None` at this point, let's add `assert build_config.additional_models is not None` before this line?